README.rst

```diff
@@ -1,4 +1,4 @@
-Home Assistant |Build Status| |CI Status| |Coverage Status| |Chat Status|
+Home Assistant |Build Status| |CI Status| |Coverage Status| |Chat Status| |Hass.io Status|
```

Comment:
We should remove coverage and travis, the are no longer used. 

I honestly think that the only badge we should keep is "Chat"